style.css

```diff
@@ -141,6 +141,24 @@ h1 {
     min-height: 500;
   }
 
+  .links {
+    cursor: pointer;
+  }
+
+  .hamburger-menu .active:first-child {
+    transform: rotateZ(28deg);
+    transform-origin: left;
+  }
+
+  .hamburger-menu .active:nth-child(2) {
+    transform: rotateZ(-28deg) translateY(-4px);
+    transform-origin: right;
+  }
+
+  .hamburger-menu .active:last-child {
+    display: none;
+  }
+
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly Work on it.

Your Project Portfolio - mobile menu Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/201697387-4db9f5f5-a154-4e7a-ba49-3398f83961f6.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/201697645-416311e8-30bf-44d1-aad1-274fdcd31fc3.PNG)